engine/src/distribution/primitives.cpp

```diff
@@ -1108,8 +150,11 @@ void sendSamplesToMaster(Context * context, const BlazingTableView & samples, st
 	auto message =
 		Factory::createSampleToNodeMaster(message_id, context_token, self_node, table_total_rows, samples);
 
+	// // Send message to master
```

Comment:
is this in another pr?

Reply:
Yep the distribution part seems incomplete so I guess will be addressed in another one